Tests/Unit/Mailer/Factory/SparkpostTransportFactoryTest.php

```diff
@@ -22,19 +23,22 @@ class SparkpostTransportFactoryTest extends TestCase
 {
     private SparkpostTransportFactory $sparkpostTransportFactory;
 
-    private TranslatorInterface|MockObject $translatorMock;
+    private TranslatorInterface&MockObject $translatorMock;
+    private CoreParametersHelper&MockObject $coreParametersHelper;
```

Comment:
We cannot use PHP's intersection types until we bump minimum PHP version to `8.1`.
It is `8.0` now.https://github.com/acquia/mc-cs-plugin-sparkpost/blob/487da41467b8f00679d1c6d1ab7de5a7ff59e355/composer.json#L16